core/scripts/vrfv2/testnet/main.go

```diff
@@ -56,6 +63,121 @@ func main() {
 	e := helpers.SetupEnv(false)
 
 	switch os.Args[1] {
+	// "generate-proof" generates a proof for a given request ID and block number
+	// using they VRF key from the given VRF node. It can optionally submit the
+	// transaction on-chain. This scripts is useful when you want to manually fulfill
+	// a request ID but do not already have a proof
+	// this script exports key from the given node and decrypts it in memory
+	// the exported key is not persisted locally
+	case "generate-proof":
+		cmd := flag.NewFlagSet("generate-proof", flag.ExitOnError)
+		vrfNodeURL := cmd.String("vrf-node-url", "", "remote node URL")
+		pubKey := cmd.String("compressed-pub-key", "", "compressed public key")
+		requestBlock := cmd.Uint64("request-block", 0, "request block")
+		requestID := cmd.String("request-id", "", "request ID")
+		coordinatorAddress := cmd.String("coordinator-address", "", "request ID")
+		executeTx := cmd.Bool("execute-tx", false, "if true, posts fulfillment transaction on-chain")
```

Comment:
What would be the use case for this script if you don't want to force fulfill the request? Just to generate the proof? Maybe we can call the script `generate-proof-and-execute-tx` (or similar) to be more clear.